src/main/kotlin/org/rust/cargo/runconfig/buildtool/CargoBuildManager.kt

```diff
@@ -73,12 +73,15 @@ object CargoBuildManager {
         val environment = buildConfiguration.environment
         val project = environment.project
 
+        // don't use `redirect input` when build
+        val oldRedirectInput = configuration.isRedirectInput
+        configuration.isRedirectInput = false
+        val config = configuration.clean().ok
+        configuration.isRedirectInput = oldRedirectInput
+        if (config == null) return CANCELED_BUILD_RESULT
```

Comment:
Can we just create a build configuration (`org.rust.cargo.runconfig.buildtool.CargoBuildManager#getBuildConfiguration`) with `isRedirectInput = false` in the same way as we force not to use sudo during the build process?

Reply:
Done.